cmd/recorder/main.go

```diff
@@ -14,27 +13,33 @@ import (
 var logger = log.Logger("lassie/event_recorder/cmd")
 
 func main() {
-	log.SetAllLoggers(log.LevelInfo)
-	ctx := context.Background()
 
 	// TODO: add flags for all options eventually.
```

Comment:
consider using urfav.cli/v2 or similar

Reply:
The code doesn't aim to provide a CLI tool. Instead, it is a service that will almost always will be running as a long-runnign process. As such, the flags are no where near complex enough to introduce big dependencies like urfave.

Will revisit as the flags grow.

Reply:
Yeah, I considered introducing urfave.cli/v2 as well, but I ultimately ended up at the same opinion of masih. The only reason there's a command is because it needs to be ran programatically. Not really made to be a CLI. If the options become way more configurable then something might be needed to help a user configure the command.